pom-step-02.xml

```diff
@@ -186,13 +186,14 @@
                 <executions>
                     <execution>
                         <id>package-jar-with-dependencies</id>
-                        <phase>prepare-package</phase>
+                        <phase>package</phase>
                         <goals>
                             <goal>single</goal>
                         </goals>
                         <configuration>
                             <finalName>openmrs-standalone</finalName>
                             <appendAssemblyId>false</appendAssemblyId>
+                            <attach>true</attach> 
```

Comment:
Why are we attaching?

Reply:
Was fetching 
```
Failed to execute goal org.apache.maven.plugins:maven-jar-plugin:3.4.1:jar (default-jar) on project referenceapplication-standalone-02: 
You have to use a classifier to attach supplemental artifacts to the project instead of replacing them.
```
Without attaching, Maven treats the assembly as a temporary build output, not an artifact, which breaks the build since it does not include `openmrs-standalone.jar` but attaching helps to treat it as the main artifact